src/Module/Conversation/Network.php

```diff
@@ -324,12 +324,12 @@ protected static function parseRequest(array $parameters, array $get)
 				DI::config()->get('system', 'itemspage_network'));
 		}
 
-		self::$min_id = $_GET['min_id'] ?? null;
-		self::$max_id = $_GET['max_id'] ?? null;
+		self::$min_id = $get['min_id'] ?? null;
+		self::$max_id = $get['max_id'] ?? null;
 
 		switch (self::$order) {
 			case 'received':
-				self::$max_id = $_GET['last_received'] ?? self::$max_id;
+				self::$max_id = $get['last_received'] ?? self::$max_id;
 				break;
 			case 'commented':
 				self::$max_id = $_GET['last_commented'] ?? self::$max_id;
```

Comment:
I hoped that you would replace the remaining references of `$_GET` in this methods, but it didn't work implicitly, so please replace the remaining references to `$_GET` in this method I couldn't reach through the GitHub review.
```suggestion
				self::$max_id = $get['last_commented'] ?? self::$max_id;
```

Reply:
Okay, done.